fractal.js

```diff
@@ -135,8 +136,8 @@ fractal.cli.command('build-site', (args, done) => {
 });
 
 web.theme(theme);
-
-web.set('static.path', path.join(__dirname, '/public'));
+// web.set('static.path', path.join(__dirname, '/dist'));
+web.set('static.path', __dirname, +'/dist');
```

Comment:
This line seems odd. I think you can change the path with:

`web.set('static.path', path.join(__dirname, '/dist'));`

Reply:
This is what it was originally, and it was breaking locally. Making the change to what's currently on 140 resolved the issue.